lib/rules/no-unexpected-multiline.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @fileoverview Rule to spot scenarios where a newline looks like it is ending a statement, but is not.
+ * @author Glen Mailer
+ * @copyright 2015 Glen Mailer
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+module.exports = function(context) {
+
+    var FUNCTION_MESSAGE = "This ( is a function call against the previous line, which is misleading.";
```

Comment:
This is a gray area. My suggestion is in line with how our other rules report problems, and I think consistency in error messages is important. Feel free to come up with another suggestion, I just dont think the current message is correct.